package.json

```diff
@@ -14,7 +14,7 @@
     "build": "tsc",
     "build:serverless": "ncc build src/index.ts -o dist",
     "start": "probot run ./lib/index.js",
-    "start:watch": "node copy.js && src/assets/images lib/assets/images && nodemon --exec 'yarn start'",
+    "start:watch": "node copy.js && nodemon --exec 'yarn start'",
```

Comment:
Why was this changed?

Reply:
`copy.js` already copies the images over and besides the second line breaks on mac so its best to remove

Reply:
not sure who added that line in the first place and why